module-4/backend/week_one.md

```diff
@@ -1,19 +1,60 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+I think completing sorting suite gave me a lot confidence. I took a udemy course on javascript over the break and then tried to do sorting suite and found that I was able to do it without too much difficulty.
+
 2. What are some tools to help debug JavaScript code?
+
+The one I use the most is console.lot, but I have gotten much more comfortable using debugger and the dev tool panels in chrome to help debug as well.
+
 3. What are some tools you need in order to unit test your JavaScript?
-4. What is the syntax for invoking a function? Give an example.
-5. What's `this` in JavaScript?
-6. What is Webpack and why is it useful?
-7. When/why do you want to use event delegation?
-8. What's `npm` and what do we use it for?
+
+You need mocha and the chai libraries.
+
+4. What is the syntax for invoking a function?
+
+You would write the name of the function (or the variable that holds the function) and then put () with parameters if the function needed them.
+
+5. What is CORS?  
```

Comment:
5, 6 are outdated questions from an old version of the repo. Please pull down the latest version